docs/advanced/analytics/getting-started.mdx

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 1
+title: Getting Started
+description:
+  Get started by learning how to track analytics events across your entire
+  application.
+---
+
+import AnalyticsFigure from "./assets/analytics.svg";
+import Figure from "@site/src/components/Figure";
+import ContactLink from "@site/src/components/ContactLink";
+import { useState } from "react";
+
+> _Since version 2.19_
+
+<p>{frontMatter.description}</p>
+
+Front-Commerce uses [`getanalytics.io`](https://getanalytics.io/) under the
```

Comment:
**issue:**  the project name is Analytics  and library is named `analytics`. I think mentioning the project web domain name instead  of its name could lead to confusion

**suggestion:** use "Analytics" or "`analytics`"
```suggestion
Front-Commerce uses [`analytics`](https://getanalytics.io/) under the
```